src/main/java/org/cryptomator/ui/fxapp/FxApplicationWindows.java

```diff
@@ -65,26 +66,26 @@ public FxApplicationWindows(@PrimaryStage Stage primaryStage, //
 								QuitComponent.Builder quitWindowBuilder, //
 								UnlockComponent.Factory unlockWorkflowFactory, //
 								UpdateReminderComponent.Factory updateReminderWindowFactory, //
-								DokanySupportEndComponent.Factory dokanySupportEndWindowBuilder, //
 								LockComponent.Factory lockWorkflowFactory, //
 								ErrorComponent.Factory errorWindowFactory, //
 								VaultOptionsComponent.Factory vaultOptionsWindow, //
 								ShareVaultComponent.Factory shareVaultWindow, //
-								ExecutorService executor) {
+								ExecutorService executor, //
+								CustomDialog.Builder customDialog) {
```

Comment:
Builders are not thread safe (i.e. if two threads at the same time build a custom dialog from FxApplicationWindows, this can lead to undefined behaviour). We should use a dagger [Provider](https://dagger.dev/dev-guide/#provider-injections).